.github/workflows/ci_workflow.yml

```diff
@@ -0,0 +1,397 @@
+  # as OS versions are retired.
+
+  # ---------------------------------------------------------------------------
+  # Linux
+  # ---------------------------------------------------------------------------
+  # Note: 'runs-on' defines the GH-hosted VM. 'container' is the Docker 
+  #       container where the build occurs.
+  # Note: 'container' images are hosted on DockerHub
+
+  linux:
+    name: Linux CentOS 7 VFX CY2019 ${{ matrix.build-desc }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        build: [1, 2, 3, 4, 5, 6]
+        include:
+          - build: 1
+            build-desc: ''
+            build-type: Release
+            python-version: 3.7
+            shared: 'ON'
+            use-sse: 'ON'
+            cxx-compiler: g++
+            cc-compiler: gcc
+          - build: 2
+            build-desc: (Debug)
+            build-type: Debug
+            python-version: 3.7
+            shared: 'ON'
+            use-sse: 'ON'
+            cxx-compiler: g++
+            cc-compiler: gcc
+          - build: 3
+            build-desc: (static, no SSE, Python 2.7)
+            build-type: Release
+            python-version: 2.7
+            shared: 'OFF'
+            use-sse: 'OFF'
+            cxx-compiler: g++
+            cc-compiler: gcc
+          - build: 4
+            build-desc: '(Clang 7)'
+            build-type: Release
+            python-version: 3.7
+            shared: 'ON'
+            use-sse: 'ON'
+            cxx-compiler: clang++
+            cc-compiler: clang
+          - build: 5
+            build-desc: (Clang 7, Debug)
+            build-type: Debug
+            python-version: 3.7
+            shared: 'ON'
+            use-sse: 'ON'
+            cxx-compiler: clang++
+            cc-compiler: clang
+          - build: 6
+            build-desc: (Clang 7, static, no SSE, Python 2.7)
+            build-type: Release
+            python-version: 2.7
+            shared: 'OFF'
+            use-sse: 'OFF'
+            cxx-compiler: clang++
+            cc-compiler: clang
+    container:
+      # TODO: Add to matrix with 'aswf/ci-ocio:2020' when available. In the 
+      #       meantime the 'Setup Python' step will install Python 3 for use
+      #       in the CY2019 container.
+      image: aswf/ci-ocio:2019
+    env:
+      CXX: ${{ matrix.cxx-compiler }}
+      CC: ${{ matrix.cc-compiler }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Python
+        run: share/ci/scripts/linux/install_python.sh 3.7.7
+        if: matrix.python-version == '3.7'
+      - name: Create build directories
+        run: |
+          mkdir _install
+          mkdir _build
+      - name: Configure
+        run: |
+          cmake ../. \
+                -DCMAKE_INSTALL_PREFIX=../_install \
+                -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} \
+                -DCMAKE_CXX_STANDARD=14 \
+                -DOCIO_BUILD_DOCS=ON \
```

Comment:
The documentation generation could be validated per platform i.e. that's not dependent of the compiler.

Reply:
Changed so that there is a doc build for each platform and Python version (so two per platform currently). I kept docs enabled for SonarCloud and one latest package build as well.